user-service/index.js

```diff
@@ -37,7 +37,8 @@ router.get('/verify-jwt', authenticateJwt, acknowledgeJWTValidity);
 
 app.use('/api/user', router).all((_, res) => {
   res.setHeader('content-type', 'application/json');

```

Comment:
Again, same here regarding the need to set headers manually

Reply:
User cannot sign in or create account after I remove the header, so I will keep this

Reply:
Again, just keep `content-type` and remove `Access-Control-Allow-Origin` and `Access-Control-Allow-Credentials`?

Reply:
It works perfectly! Thanks for spotting it 👍